C24_WMDE_Desktop_DE_02

Sperling-0 opened the pull request:
Ticket: https://phabricator.wikimedia.org/T368509

Abban commented on mobile/components/Slides.jsx:
```diff
@@ -1,28 +1,31 @@
-	return <div className="mini-banner-carousel">
+export default function Slides( { formattedGoalDonationSumNumeric, campaignDaySentence, currentDayName, progressBar, visitorsVsDonorsSentence } ) {
+
+	return <div><div className="mini-banner-carousel">
```
What's the extra div for?

moiikana replied:
relicts of bygone times, removed them

Sperling-0 committed:
Prepare campaign for C24_WMDE_Desktop_DE_02

Ticket: https://phabricator.wikimedia.org/T368509

Sperling-0 committed:
Prepare CTRL for C24_WMDE_Desktop_DE_02

- Update tests
- Fix bug in slide show content

Ticket: https://phabricator.wikimedia.org/T368509

Co-authored by: Abban Dunne <abban.dunne@wikimedia.de>

Sperling-0 committed:
Implement VAR for C24_WMDE_Desktop_DE_02

- A bigger headline is added to the VAR banner
- The (i) icon is moved to the bigger headline and increased in size
- Both banners share the same height

Ticket: https://phabricator.wikimedia.org/T368509

moiikana commented on src/themes/Svingle/Message/Message.scss:
```diff
@@ -26,5 +26,31 @@ $font-sizes: ( breakpoints.$extra-small: 18px ) !default;
+
+		&-title {
+			max-height: 62px;
+			margin-bottom: 4px !important;
```
created a followup PR with explanations here:
https://github.com/wmde/fundraising-banners/pull/493

moiikana reviewed (changes requested):
these changes should be adapted: https://github.com/wmde/fundraising-banners/pull/493

moiikana committed:
[FollowUp] Remove !important rule (#493)

- !important in CSS is usually a sign of a code smell when writing CSS,
because it means there might was trouble with the CSS's specificity rules (
see e.g. https://csswizardry.com/2012/11/code-smells-in-css/ -> "Reactive !important")

- exchange the p tag (which have strong rather global css rules) with a normal div tag
(that is easier to style, we don't have to overwrite p rules)

moiikana commented on src/themes/Svingle/Message/Message.scss:
```diff
@@ -26,5 +26,31 @@ $font-sizes: ( breakpoints.$extra-small: 18px ) !default;
+			max-height: 62px;
+			margin-bottom: 4px !important;
+			text-align: center;
+			font-size: 25px;
```
should be `font-size: 1.4em;`,
else the text will always have a fixed size (25px).

If you don't remember this, please read up again on Relative length units:
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units

moiikana commented on src/themes/Svingle/Slider/KeenSlider.scss:
```diff
@@ -48,6 +48,33 @@ $margin-bottom: 15px !default;
+			max-height: 62px;
+			margin: 0 0 4px;
+			text-align: center;
+			font-size: 25px;
```
1.4em here as well
